metadata/src/test/java/org/apache/kafka/controller/QuorumFeaturesTest.java

```diff
@@ -74,7 +74,7 @@ public void testDefaultFeatureMapWithUnstable() {
         for (Features feature : Features.PRODUCTION_FEATURES) {
             expectedFeatures.put(feature.featureName(), VersionRange.of(
                 0,
-                feature.defaultValue(MetadataVersion.latestTesting())
+                feature.defaultValue(MetadataVersion.LATEST_PRODUCTION)
```

Comment:
Hmm, why is this changed to LATEST_PRODUCTION? The test is about unstable features.